Automattic/jetpack
# Podcast Player: Use CSS mask to color ME.js controls

Origin: https://github.com/Automattic/jetpack/issues/15235#issuecomment-607808472

#### Changes proposed in this Pull Request:
* Support custom colors for `MediaElement`'s controls via the `background-color` property (instead of interpolating SVG URL string with encoded values).

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* This adds a possibility to customize `MediaElement`'s colors via CSS variables, as discussed in https://github.com/Automattic/jetpack/issues/15235. 
* IE11 remains supported as it was.

#### Testing instructions:
* There should be no visible change for the Podcast Player block between this branch and the `feature/podcast-player` branch.
* To test customization of controls colors (in modern browsers), go to the bottom of [`styles.css`](https://github.com/Automattic/jetpack/compare/feature/podcast-player...try/podcast-player-mejs-svg-colors?expand=1#diff-23936c23c59259d544403d0cc941ea1d) and set any color to the [background-color property of the ME.js button](https://github.com/Automattic/jetpack/compare/feature/podcast-player...try/podcast-player-mejs-svg-colors?expand=1#diff-23936c23c59259d544403d0cc941ea1dR340). Refresh the page and see if the color has been set. Play with the controls by clicking them - it should toggle colored `play`/`pause`, `mute`/`unmute` and `replay` (thanks, @frontdevde!) SVGs.

#### Proposed changelog entry for your changes:

* Podcast Player: Use CSS mask to color ME.js controls
